SpherePacking/ModularForms/summable_lems.lean

```diff
@@ -694,7 +693,7 @@ lemma G_2_alt_summable (z : ℍ) : Summable fun  (m : Fin 2 → ℤ) =>
       (Real.rpow_nonneg (norm_nonneg ![b 0, b 1 + 1]) (-1))
 
 
-def δ (a b : ℤ): ℂ := if a = 0 ∧ b = 0 then 1 else if a = 0 ∧ b = -1 then 2 else 0
+noncomputable def δ (a b : ℤ): ℂ := if a = 0 ∧ b = 0 then 1 else if a = 0 ∧ b = -1 then 2 else 0
```

Comment:
this one is really surprising!! will report elsewhere

Reply:
This is the error I got: 

> failed to compile definition, compiler IR check failed at 'δ._closed_4'. Error: depends on declaration 'Nat.cast._at_.Complex.exp'.spec_1', which has no executable code; consider marking definition as 'noncomputable'

Reply:
Bizarre!